src/transport/raw/BLE.h

```diff
@@ -48,8 +48,17 @@ class BleListenParameters
 
     Ble::BleLayer * GetBleLayer() const { return mLayer; }
 
+    bool HasOverrideExistingTransport() const { return mOverrideExistingTransport; }
```

Comment:
Please find some naming that makes it clearer from reading code on what we are overriding and with what.

Maybe invert and call `PreserveExistingBleLayerTransport`.

Add doc-comments on when one would want to override or preserve the BLE layer transport. Is  this class intended to work at all when ble layer transport is not overriden? If we do not override the transport, I would expect any received BLE packets to not be received, so the transport does not fully work.

Reply:
In hindsight, depending on sideffect, I believe not overriding make the transport write-only (since no packet received callbacks). Is that true? documentation is a must.

Reply:
Fixed naming in https://github.com/project-chip/connectedhomeip/pull/15525 and added comments explaining rationale and behavior.